docs/source/user_guide/server_context.rst

```diff
@@ -0,0 +1,120 @@
+.. _user_guide_server_context:
+
+==============
+Server context
+==============
+
+What is server context
+----------------------
+
+The :class:`ServerContext <ansys.dpf.core.server_context>` class drives the
+default capabilities a server will be started with. 
+
+The server context is composed of the following information: 
+- context_type, :class:`LicensingContextType <ansys.dpf.core.server_context>`
+class object, that defines if a License checkout is required or not.
+- the xml_path that sets DPF default operators capabilities. For more
+information, see :class:`AvailableServerContexts <ansys.dpf.core.server_context>`
+and :ref:`user_guide_xmlfiles`.
+
+Two main licensing context type capabilities are available: 
+- Entry (default):  Loads the minimum capabilities without requiring 
+any license checkout.
+- Premium: Enables the entry capabilities and the capabilities that
+requires a license checkout. More operators are available.
+
+The operators list for each licensing context type is available at
+:ref:`ref_dpf_operators_reference`.
+
+Getting started with Entry capabilities
+---------------------------------------
+
+Find the list of operators available when the context is Entry at :ref:`ref_dpf_operators_reference`.
```

Comment:
to coordinate with @PProfizi

Reply:
It should be fine, I did not change the reference name